nucleus/dataset.py

```diff
@@ -795,6 +794,7 @@ def _append_video_scenes(
             )
 
         for scene in scenes:
+            scene.use_privacy_mode = self.use_privacy_mode
             scene.validate()
```

Comment:
adding this property from dataset -> scene simplifies validation later on, w/o having to pass parameters everywhere